src/Interpreter.js

```diff
@@ -17,6 +17,7 @@ export default class Interpreter {
     programCounter: number;
     memory: { [string]: any };
     isRunning: boolean;
+    timeDelayMs: number;
     onRunningStateChange: (InterpreterRunningState) => void;
 
     constructor(onRunningStateChange: (InterpreterRunningState) => void) {
```

Comment:
Add `stepTimeMs` as a parameter to the constructor and use this to initialize `this.stepTimeMs`. So that the starting value can be passed from the outside (`App`).